martin/src/config/file/tiles/postgres/builder.rs

```diff
@@ -335,6 +342,13 @@ impl PostgresAutoDiscoveryBuilder {
         let source = PostgresSource::new(id, sql_info, tilejson, self.pool.clone());
         sources.push(Box::new(source));
     }
+
+    fn configured_tables(&self) -> Vec<(&str, &str)> {
```

Comment:
Does not make a performance difference, unless number of configured table is pretty insane.

We want to only check for membership => lets choose a hashset

```suggestion
    fn configured_tables(&self) -> HashSet<(&str, &str)> {
```